tests/integration/error_warning_file_tests.py

```diff
@@ -105,39 +105,39 @@ def setUpClass(cls):
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas4 = TASFactory(account_num=4, allocation_transfer_agency=None, agency_identifier='028',
                               beginning_period_of_availa='2010', ending_period_of_availabil='2011',
                               availability_type_code=None, main_account_code='0406', sub_account_code='000',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas5 = TASFactory(account_num=5, allocation_transfer_agency='069', agency_identifier='013',
                               beginning_period_of_availa=None, ending_period_of_availabil=None,
                               availability_type_code='X', main_account_code='2050', sub_account_code='005',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas6 = TASFactory(account_num=6, allocation_transfer_agency='028', agency_identifier='028',
                               beginning_period_of_availa=None, ending_period_of_availabil=None,
                               availability_type_code='X', main_account_code='8007', sub_account_code='000',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas7 = TASFactory(account_num=7, allocation_transfer_agency=None, agency_identifier='049',
                               beginning_period_of_availa=None, ending_period_of_availabil=None,
                               availability_type_code='X', main_account_code='0100', sub_account_code='000',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas8 = TASFactory(account_num=8, allocation_transfer_agency=None, agency_identifier='049',
                               beginning_period_of_availa='2010', ending_period_of_availabil='2011',
                               availability_type_code=None, main_account_code='0100', sub_account_code='000',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
             tas9 = TASFactory(account_num=9, allocation_transfer_agency=None, agency_identifier='049',
                               beginning_period_of_availa='2014', ending_period_of_availabil='2015',
                               availability_type_code=None, main_account_code='0100', sub_account_code='000',
-                              internal_start_date='01-01-2000')
+                              internal_start_date='01-01-2000', financial_indicator2='F')
```

Comment:
Are we trying to make it ignore all of these rows? Because having a financial indicator of F will make it ignore the row entirely.

Reply:
Yup, the point of this test is to see the error warning reports are being properly generated so this just negates A33 checks which weren't in the file in the first place.